src/main/java/at/ac/fhcampuswien/fhmdb/HomeController.java

```diff
@@ -63,4 +72,19 @@ public void initialize(URL url, ResourceBundle resourceBundle) {
 
 
     }
+
+    /**
+     * This method performs a search operation on a list of movies based on a given search term.
+     * It updates the observableMovies list with the search results.
+     *
+     * @param searchTerm The keyword to search in the movie titles and descriptions.
+     */
+    private void searchKeyword(String searchTerm) {
+        observableMovies.setAll(allMovies);
+        MovieSearchService movieSearchService = new MovieSearchService(observableMovies);
```

Comment:
I would move this `MovieSearchService` outside of the method and make it a field in the class which gets instantiated in the `initialize()` method, after line 44

Reply:
Updated based on the recommendation